.github/workflows/build-crates-individually.yml

```diff
@@ -52,13 +52,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3.5.2
 
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          profile: minimal
-          override: true
+      # Setup Rust with stable toolchain and minimal profile
+      - name: Setup Rust
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=stable --profile=minimal
```

Comment:
We repeat this same script a lot, should it be a shell script? Or would that be more complicated?

I'm concerned that these duplicate scripts might diverge over time, because we might miss some when we do fixes or changes. The action was only a name and a version, so it was easier to keep in sync.

Reply:
Yes, I'll do this globally with: 
- https://github.com/ZcashFoundation/zebra/issues/6168